.buildkite/scripts/create_new_rc.sh

```diff
@@ -12,6 +12,7 @@ source .buildkite/shared/lib/rc.sh
 # repository.
 readonly -a STACKS=(
     grapl/testing
+    grapl/integration-tests/testing
```

Comment:
Based on how the [similar code](https://github.com/grapl-security/pipeline-infrastructure/blob/7d8ae5f93b9ae08b99fde70fe4246c178ee2373d/.buildkite/scripts/create_new_rc.sh#L15-L20) works in `pipeline-infrastructure`, I suspect this needs to be `integration-tests/testing`.

I really need to extract this code to a plugin 🤦